src/Razor/src/Microsoft.AspNetCore.Razor.Language/Components/ComponentCodeDirective.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Razor.Language.Components
+{
+    public static class ComponentCodeDirective
```

Comment:
Name this `CodeDirective` and the `Register` below as `RegisterForComponents`

Reply:
I'm just following the same pattern as other component directives.

Reply:
Ah I see, didn't realize we did that. Leave for now then, something we should revisit for all of the component like directives before RTM